daemon/mgr/container_utils.go

```diff
@@ -268,6 +268,13 @@ func validateResource(r *types.Resources, update bool) ([]string, error) {
 			warnings = append(warnings, MemorySwapWarn)
 			r.MemorySwap = 0
 		}
+		if r.Memory > 0 && r.MemorySwap > 0 && r.MemorySwap < r.Memory {
+			return warnings, fmt.Errorf("Minimum memoryswap limit should be larger than memory limit")
+		}
+		// cgroup not allow memory-swap less than memory limit
```

Comment:
The comment is for the `if r.Memory > 0 && r.MemorySwap > 0 && r.MemorySwap < r.Memory {`?